src/OpenTok/OpenTok.php

```diff
@@ -385,6 +385,87 @@ public function listArchives($offset=0, $count=null)
         return new ArchiveList($archiveListData, array( 'client' => $this->client ));
     }
 
+    /**
+     * Initiate an outgoing SIP call
+     *
+     * @param string $sessionId The OpenTok SessionIdwhere the participant being called
+     * will join.
+     *
+     * @param string $token The token for conecting to an OpenTok session. This is the same
+     * as the one created by Session.generateToken.
+     *
+     * @param string $username Auth​: Username to be used in the SIP INVITE request for HTTP
```

Comment:
these docs are outdated, the username and password are part of options now. @jeffswartz will have to do a final pass anyway so i'm fine with merging this now.